react/IntentIframe/IntentIframe.jsx

```diff
@@ -17,7 +19,13 @@ class IntentIframe extends React.Component {
   componentDidMount() {
     const { action, data, type, onCancel, onError, onTerminate } = this.props
```

Comment:
Maybe we should add `client` in those props

Reply:
Yes, the purpose of withClient is to add the client in the props of the underlying component. You're right that it's better to destructure the client at the top of the function 👍